test/IonParserBinaryRaw.ts

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
> Do we still need the JS variant of the tests?

Yes, the new mocha tests are not yet part of the build/release targets for grunt. I'm going to remove all of the intern.js tests in a PR after the mocha tests are complete. Otherwise I'd need to change the build logic to run both (partial) tests suites in the interim.

Reply:
> _Minor:_ (c) 2019

My understanding is that the copyright notice year needs to be the first year it was covered, not the current year, allowing us to avoid an annual sweep/update process.

Reply:
Oh okay, that seems reasonable to me.